docs/pages/modules/autolinking.mdx

```diff
@@ -204,9 +204,10 @@ All projects created with the `npx create-expo-app` command are already configur
 The module resolution algorithm searches only for packages that contain the [Expo module config](/modules/module-config/) file (**expo-module.config.json**) at the root directory, next to the **package.json** file.
 It's also necessary to include supported platforms in the `platforms` array — if the platform for which the autolinking algorithm is run is not present in this array, it's just skipped in the search results.
 
-### How is it different from React Native autolinking?
+### How is it different from React Native community autolinking?
```

Comment:
```suggestion
### How is it different from React Native community CLI autolinking?
```

Small suggestion: Let's use the term that a user is most familiar with here to help them. I am assuming that the autolinking is a feature of the community CLI rather than react-native package, so it's better to use the complete phrase here.

Reply:
Sure, sounds good to me